CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 ## Bug Fixes
 + Support `useCallback` with a function that takes arguments [PR #290](https://github.com/shadaj/slinky/pull/290)
 
+### Bug Fixes
```

Comment:
Bad merge here, you can move my bug fix item under yours since your `###` heading is correct.